packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js

```diff
@@ -23,26 +20,23 @@ export default {
     knobs: {
       VideoPlayer: ({ groupId }) => ({
         showCaption: boolean('Show caption (showCaption):', true, groupId),
-        aspectRatio: select(
-          'Aspect ratio (aspectRatio):',
-          ratios,
-          ratios[0],
-          groupId
-        ),
+        aspectRatio: 'default',
+        videoId: '0_uka1msg4',
       }),
     },
```

Comment:
Given different story seems to have different set of parameters now, would you move this block to the `parameters` of `Default` story?

Reply:
I'm sorry, @asudoh, I didn't understand quite well your question. 😭

Reply:
No problem @guilhermelMoraes, I meant:

```diff
diff --git a/packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js b/packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js
index d7589859..50aa4dd2 100644
--- a/packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js
+++ b/packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js
@@ -17,13 +17,6 @@ export default {
     percy: {
       skip: true,
     },
-    knobs: {
-      VideoPlayer: ({ groupId }) => ({
-        showCaption: boolean('Show caption (showCaption):', true, groupId),
-        aspectRatio: 'default',
-        videoId: '0_uka1msg4',
-      }),
-    },
   },
 };
 
@@ -46,6 +39,18 @@ export const Default = ({ parameters }) => {
   );
 };
 
+Default.story = {
+  parameters: {
+    knobs: {
+      VideoPlayer: ({ groupId }) => ({
+        showCaption: boolean('Show caption (showCaption):', true, groupId),
+        aspectRatio: 'default',
+        videoId: '0_uka1msg4',
+      }),
+    },
+  }
+}
+
 export const aspectRatio1x1 = ({ parameters }) => {
   return <Default parameters={parameters} />;
 };
```

Reply:
@guilhermelMoraes, I think Akira is saying that you need to do for Default story the same thing you did for aspectRatio1x1 and 4x3 story.

e.g.
~~~js
aspectRatio1x1.story = {
  name: 'Aspect ratio 1:1',
  parameters: {
    props: {
      VideoPlayer: () => {
        return {
          showCaption: true,
          aspectRatio: '1x1',
          videoId: '1_9h94wo6b',
        };
      },
    },
  },
};
~~~

cc: @asudoh

Reply:
Let me explain more on this; The parameters defined in the default export affects all stories. Given I think we don't want that, we can move it to the `Default` story. Note that the default export and the default story are different animals; The former is for metadata for all stories, the latter is for a story with "default" name.

Reply:
Yeah, got it!! Thanks @asudoh

Reply:
Isn't necessary to follow same ideia on props/knobs for Default story as aspectRatio1x1 and 4x3 ? @asudoh

Reply:
thanks for the clarification, @asudoh, and @diegomarcuz. ;)